git_machete/cli.py

```diff
@@ -45,304 +44,517 @@
 ]
 
 
-def usage(command: str = None) -> None:
-    if command and command in command_by_alias:
-        command = command_by_alias[command]
+def get_help_description(command: str = None) -> str:
+    usage_str = ''
     if command and command in long_docs:
-        print(fmt(textwrap.dedent(long_docs[command])))
+        usage_str += fmt(textwrap.dedent(long_docs[command]))
     else:
-        print()
-        short_usage()
+        usage_str += get_short_general_usage() + '\n'
         if command:
-            print(f"\nUnknown command: '{command}'")
-        print(fmt("\n<u>TL;DR tip</u>\n\n"
-                  "    Get familiar with the help for <b>format</b>, <b>edit</b>,"
-                  " <b>status</b> and <b>update</b>, in this order.\n"))
+            usage_str += f"\nUnknown command: '{command}'"
+        usage_str += fmt(
+            "\n<u>TL;DR tip</u>\n\n"
+            "    Get familiar with the help for <b>format</b>, <b>edit</b>,"
+            " <b>status</b> and <b>update</b>, in this order.\n\n")
         for hdr, cmds in command_groups:
-            print(underline(hdr))
-            print()
+            usage_str += underline(hdr) + '\n\n'
             for cm in cmds:
                 alias = f", {alias_by_command[cm]}" if cm in alias_by_command else ""
-                print("    %s%-18s%s%s" % (EscapeCodes.BOLD, cm + alias, EscapeCodes.ENDC, short_docs[
-                    cm]))  # bold(...) can't be used here due to the %-18s format specifier
-            sys.stdout.write("\n")
-        print(fmt(textwrap.dedent("""
+                usage_str += ("    %s%-18s%s%s" % (
+                    # bold(...) can't be used here due to the %-18s format specifier
+                    EscapeCodes.BOLD, cm + alias, EscapeCodes.ENDC, short_docs[cm]))
+                usage_str += '\n'
+            usage_str += '\n'
+        usage_str += fmt(textwrap.dedent("""
             <u>General options</u>\n
                 <b>--debug</b>           Log detailed diagnostic info, including outputs of the executed git commands.
                 <b>-h, --help</b>        Print help and exit.
                 <b>-v, --verbose</b>     Log the executed git commands.
                 <b>--version</b>         Print version and exit.
-        """[1:])))
+        """[1:]))
+    return usage_str
 
 
-def short_usage() -> None:
-    print(
-        fmt("<b>Usage: git machete [--debug] [-h] [-v|--verbose] [--version] "
-            "<command> [command-specific options] [command-specific argument]</b>"))
+def get_short_general_usage() -> str:
+    return (fmt("<b>Usage: git machete [--debug] [-h] [-v|--verbose] [--version] "
```

Comment:
Nit: in error messages, the program name is `git-machete`, can be changed to `git machete`:
```
$ git machete a
usage: git-machete [--debug] [-h] [--version] [-v]
```

Reply:
Although not necessary if problematic, since both `git machete` and `git-machete` are both valid invocations; the former is just nicer-looking xD